server/models/game.go

```diff
@@ -30,12 +32,21 @@ type Player struct {
 	Points uint64 `json:"points"`
 }
 
+type Prompts struct {
```

Comment:
Not: what do you think about prompt singular?

Reply:
Just change the name "Prompts" to "Prompt"? Or do you mean change the struct field to just hold a single prompt instead of three? Then we could have a "Prompt Type" to identify if the prompt is noun, adjective, etc.  This would allow more specific future types like "Person", "Place", "Vegetable" for future game experiences / modes.